src/index.js

```diff
@@ -1,4 +1,5 @@
-import 'babel-polyfill';
+/* eslint-disable simple-import-sort/imports */
```

Comment:
maybe add a comment that explains why imports have to placed in that particular order?